src/Compilers/VisualBasic/Portable/Operations/VisualBasicOperationFactory.vb

```diff
@@ -962,7 +1009,12 @@ Namespace Microsoft.CodeAnalysis.Operations
             Dim type As ITypeSymbol = Nothing
             Dim constantValue As [Optional](Of Object) = New [Optional](Of Object)()
             Dim isImplicit As Boolean = boundCaseBlock.WasCompilerGenerated
-            Return New LazySwitchCase(ImmutableArray(Of ILocalSymbol).Empty, clauses, body, _semanticModel, syntax, type, constantValue, isImplicit)
+
+            Dim condition As Lazy(Of IOperation) = If(boundCaseBlock.CaseStatement.ConditionOpt Is Nothing,
+                                                      OperationFactory.NullOperation,
+                                                      New Lazy(Of IOperation)(Function() Create(boundCaseBlock.CaseStatement.ConditionOpt)))
```

Comment:
`If(expr Is Nothing, OperationFactory.NullOperation, New Lazy(Of IOperation)(Function() Create(expr)))` is used here while `New Lazy(Of IOperation)(Function() Create(expr))` is used in most other cases. We should be consistent, ideally with a `CreateLazyOperation(exprOpt As BoundExpression)` helper that returns `OperationFactory.NullOperation` if `exprOpt Is Nothing`.

Reply:
Sure, will switch to ```New Lazy(Of IOperation)(Function() Create(boundCaseBlock.CaseStatement.ConditionOpt))```  

---
In reply to: [184538767](https://github.com/dotnet/roslyn/pull/26395#discussion_r184538767) [](ancestors = 184538767)